pkg/kubernetes/config/kubeconfig.go

```diff
@@ -151,12 +151,16 @@ func RenameConfig(sp *specs.Specs, newConfig *clientcmdapi.Config) {
 	newConfig.Clusters[sp.GetClusterName()] = newConfig.Clusters[DefaultClusterName]
 	delete(newConfig.Clusters, DefaultClusterName)
 
+	log.Debug("Renaming user")
+	newConfig.AuthInfos[sp.GetClusterName()] = newConfig.AuthInfos[DefaultClusterAdminName]
```

Comment:
Let's add a prefix of `wks-` to the values.  That way the user will know what clusters were created with wksctl when looking in their config file.

Reply:
👍